core/src/main/java/org/kiwix/kiwixmobile/core/bookmark/BookmarksContract.kt

```diff
@@ -15,24 +15,27 @@
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  *
  */
+package org.kiwix.kiwixmobile.core.bookmark
 
-package org.kiwix.kiwixmobile.core.bookmark;
+import org.kiwix.kiwixmobile.core.base.BaseContract
+import org.kiwix.kiwixmobile.core.base.BaseContract.Presenter
+import org.kiwix.kiwixmobile.core.base.BaseContract.View
 
-import java.util.List;
-import org.kiwix.kiwixmobile.core.base.BaseContract;
-
-interface BookmarksContract {
-  interface View extends BaseContract.View<Presenter> {
-    void updateBookmarksList(List<BookmarkItem> bookmarks);
-
-    void notifyBookmarksListFiltered(List<BookmarkItem> bookmarks);
+internal interface BookmarksContract {
+  interface View :
+    BaseContract.View<Presenter?> {
+    fun updateBookmarksList(bookmarks: List<BookmarkItem?>?)
+    fun notifyBookmarksListFiltered(bookmarks: List<BookmarkItem?>?)
   }
 
-  interface Presenter extends BaseContract.Presenter<View> {
-    void loadBookmarks(boolean showBookmarksCurrentBook);
-
-    void filterBookmarks(List<BookmarkItem> bookmarksList, String newText);
+  interface Presenter :
+    BaseContract.Presenter<View?> {
```

Comment:
Shouldn't this be on the same lines? Or is this being handled by our code formatting tool?

Reply:
I did use `ctrl + alt + l` in order to reformat the code after the automatic conversion to Kotlin. Nothing was changed. So I think it's being handled by the code formatting tool

Reply:
No, the automatic conversion just gets stupid sometimes and breaks all crazily. It can go on the same line